.gitignore

```diff
@@ -1,5 +1,5 @@
 ## Ignore WebConfig because it contains sensitive values
-Web.config
+Authy2FA/Web.config
```

Comment:
You're right @devinrader, usually we add an instruction in the README project like this one:

_Create a copy of BrowserCalls.Web/Web.config.sample and rename it to BrowserCalls.Web/Web.config_.

e.g. [Browser Calls (ASP.NET MVC)#Local Development](https://github.com/TwilioDevEd/browser-calls-csharp#local-development)